nixos/modules/virtualisation/containers.nix

```diff
@@ -799,6 +811,12 @@ in
                 LOCAL_ADDRESS6=${cfg.localAddress6}
               ''}
             ''}
+            ${optionalString (cfg.writableAPIVFS == true) ''
+              SYSTEMD_NSPAWN_API_VFS_WRITABLE=1
+            ''}
+            ${optionalString (cfg.writableAPIVFS == "network") ''
```

Comment:
So,are there any other possible values?

Reply:
No. The values are constrained by the enum [here](https://github.com/NixOS/nixpkgs/pull/73467/files#diff-46d38a04634eee9a218f83f4623f0c31R568), and this encodes the full set of allowed values as outlined in the documentation [here](https://github.com/NixOS/nixpkgs/pull/73467/files#diff-46d38a04634eee9a218f83f4623f0c31R568).

Reply:
I meant if SYSTEMD_NSPAWN_API_VFS_WRITABLE allowed any other value

Reply:
No, as explained in the systemd [docs](https://systemd.io/ENVIRONMENT.html) mentioned in the PR description.